File: src/main/java/com/huang/pojo/Consumer.java
package com.huang.pojo;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class Consumer {
    @TableId(type = IdType.AUTO)
    public Integer userid;
    public String username;
    public String password;
}

File: src/main/kotlin/com/huang/pojo/Consumer.kt
package com.huang.pojo

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * 顾客表
 */
@ApiModel("顾客模型")
data class Consumer(
    @ApiModelProperty("顾客ID")
    @TableId(type = IdType.AUTO)
    var userid: Long?,

    @ApiModelProperty("用户名")
    var username: String,

    @ApiModelProperty("密码")
    var password: String
)
